components/brave_wallet_ui/constants/types.ts

```diff
@@ -116,6 +116,7 @@ export type PanelTypes =
   | 'activity' // Transactions
   | 'currencies'
   | 'transactionStatus'
+  | 'siweError'
```

Comment:
do we need a new panel type or can we just show the error panel if the error is dectected?

Reply:
Removed